doc/htmldoc/networks/spatially_structured_networks.rst

```diff
@@ -1311,9 +1311,15 @@ The following table presents some query functions provided by NEST.
 | ``nest.GetTargetNodes()``       | Obtain targets of sources in a                      |
 |                                 | given target layer.                                 |
 +---------------------------------+-----------------------------------------------------+
+| ``nest.GetSourceNodes()``       | Obtain sources of targets in a                      |
```

Comment:
Here an in corresponding locations below, I suggest to order alphabetically, i.e., GetSourceNodes before GetTargetNodes

Reply:
Done in 599eafd.